pylint/utils.py

```diff
@@ -1157,6 +1157,12 @@ def _ini_format(stream, options, encoding):
             print('#%s=' % optname, file=stream)
         else:
             value = _encode(value, encoding).strip()
+            if section == 'MESSAGES CONTROL' and ',' in str(value):
```

Comment:
I agree that it is too specialized. Is it better if we generate multiple line output only if the value matches the regex ``\([a-zA-z0-9-_]+,\)+[a-zA-z0-9-_]+``?